content/news/2021-01-08-mixxx-to-the-first-time-contributor.md

```diff
@@ -0,0 +1,110 @@
+title: Mixxx to the first time contributor
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-08 17:38:00
+summary: Before the Outreachy internship program, I had never contributed to open-source software. I didn’t know what the term "open source" meant or how different it was from other software. Looking back, it's hard to believe how clueless I was four months ago and then compare to how far I've come. I think it is safe to say that I have learned a lot about different open source software, Mixxx in particular, and the different ways that I can contribute. I feel very fortunate to have been offered the Outreachy internship, and now I will share everything that I know about contributing to the Mixxx open source projects - from the perspective of a first time contributor :)
+
+![Mixxx logo]({static}/images/news/mixxx_logo.png)
+
+Mixxx is a free/open source DJ performance software for Windows/Mac/and Linux. It was first released in 2003 as a graduate student project, and was picked up by the community and greatly enhanced. All contributors, from programmers to skin designers to translators are unpaid volunteers, hence it is entirely community driven.
```

Comment:
This comes a bit out of the blue. It's not clear what this blogpost is about. Especially the first sentence should be clear to almost everyone that reads our website ;-)

Reply:
Alright. Let me work on that.